src/megatron/bridge/data/hf_processors/openmathinstruct2.py

```
@@ -0,0 +1,58 @@
+OpenMathInstruct-2 contains math problems with generated solutions. Each example
+has ``problem``, ``generated_solution``, and ``expected_answer`` fields.
+"""
+
+from typing import Any, Optional
+
+from megatron.bridge.data.builders.hf_dataset import ProcessExampleOutput
+from megatron.bridge.training.tokenizers.tokenizer import MegatronTokenizer
+
+
+def process_openmathinstruct2_example(
+    example: dict[str, Any], tokenizer: Optional[MegatronTokenizer] = None
```

Ruff (0.15.5)

[warning] Unused function argument: `tokenizer`

(ARG001)